src/page-modules/assistant/server/journey-planner/index.ts

```diff
@@ -227,7 +227,10 @@ export function createJourneyApi(
         TripsWithDetailsQueryVariables
       >({
         query: TripsWithDetailsDocument,
-        variables: tripQuery.query,
+        variables: {
+          ...tripQuery.query,
+          arriveBy: false,
```

Comment:
Never makes sense that this can be true?

Reply:
No, because we are looking up a specific trip for a specific departure time. This means we want to find a trip that departs by time xx, if that makes sense?

Reply:
But to have the correct search mode when navigating back to the trip search results from trip details we have to know if the search was done as `arriveBy` or `departBy` and that's why I modified the compressed query

Reply:
👍